x-pack/plugins/reporting/public/share_context_menu/reporting_panel_content/reporting_panel_content.tsx

```diff
@@ -84,11 +84,11 @@ class ReportingPanelContentUi extends Component<Props, State> {
   }
 
   private getAbsoluteReportGenerationUrl = (props: Props) => {
-    const relativePath = this.props.apiClient.getReportingJobPath(
+    const relativePath = this.props.apiClient.getReportingPublicJobPath(
       props.reportType,
       this.props.apiClient.getDecoratedJobParams(this.props.getJobParams(true))
     );
-    return url.resolve(window.location.href, relativePath); // FIXME: '(from: string, to: string): string' is deprecated
```

Comment:
I can't figure out why my editor said this line uses a deprecated signature, so the comment isn't valid

Reply:
Nice I'm also not seeing any issues in my editor 👍🏻